website/index.html

```diff
@@ -5,6 +5,7 @@
   <head>
     <title>quick-lint-js finds JavaScript bugs</title>
     <meta charset="utf-8" />
+		<meta name="description" content="quick-lint-js gives you instant feedback as you code. Find bugs in your JavaScript before your finger leaves the keyboard."/>
```

Comment:
Nit: Run `yarn fmt` to reformat the code. (It looks like the indentation is way off, for example.)